portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -83,13 +83,14 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 
 		<c:if test="<%= deltaConfigurable %>">
 			<div class="dropdown pagination-items-per-page">
-				<button aria-describedby="<%= ariaPaginationResults %>" aria-haspopup="true" class="dropdown-toggle page-link" data-toggle="liferay-dropdown">
+				
+				<button aria-labelledby="<%= ariaPaginationResults %>" aria-expanded="false" aria-controls="pagination-picker" aria-haspopup="listbox" class="dropdown-toggle page-link" data-attribute="<%= delta %>" data-toggle="liferay-dropdown" role="combobox">
 					<liferay-ui:message arguments="<%= delta %>" key="x-entries" /><span class="sr-only"><%= StringPool.NBSP %><liferay-ui:message key="per-page" /></span>
 
 					<aui:icon image="caret-double-l" markupView="lexicon" />
 				</button>
 
-				<ul class="dropdown-menu dropdown-menu-top">
+				<ul class="dropdown-menu dropdown-menu-top" id="pagination-picker" role="listbox" tabindex="-1">
```

Comment:
Be careful to follow the existing patterns of the file you're working in. Notice the Id declared on on line 81. It is namespaced and follows a camelCased format. The namespace is necessary since IDs have to be unique and we could have multiple pagination components on the same page. This ID could be declared up in the same block and then used in both places.

Reply:
My mistake, I'll fix that. 
Thanks @ethib137 !